src/main/resources/anniv5Resources/localization/eng/serpentinepack/Cardstrings.json

```diff
@@ -0,0 +1,36 @@
+{
+"${ModID}:ApplyPoison": {
+    "NAME": "Apply Poison",
```

Comment:
Kind of an awkward name since this is an action on a card. Would appreciate renaming it. Something like poison Coat(ing) maybe?